Ladybird/Android/src/main/java/org/serenityos/ladybird/LadybirdActivity.kt

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2023, Andrew Kaster <akaster@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+package org.serenityos.ladybird
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.TextView
+import org.serenityos.ladybird.databinding.ActivityMainBinding
+import org.serenityos.ladybird.TransferAssets
+
+class LadybirdActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
+    private lateinit var resourceDir: String
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        resourceDir = TransferAssets.transferAssets(this)
+        initNativeCode(resourceDir)
+        view = WebViewImplementationNative()
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        view.dispose()
+    }
+
+    private lateinit var view : WebViewImplementationNative
```

Comment:
```suggestion
    private lateinit var view: WebViewImplementationNative
```

Reply:
Is there a common Kotlin linter I can enable in CI?

Reply:
I haven't used them, but apparently:
https://pinterest.github.io/ktlint/
https://detekt.dev/

Reply:
Hm. I ran the files through the builtin Android Studio linter. Adding a CI-able linter seems like something to do at the same time that we move the CI job from invoking cmake manually to invoking gradle.